tests/scripts/thread-cert/command.py

```diff
@@ -30,11 +30,13 @@
 import sys
 
 import ipv6
-from network_data import Prefix, BorderRouter, LowpanId
+import network_data
 import network_layer
 import config
 import mle
 
+from binascii import hexlify
```

Comment:
Merge this block wtih imort sys

Reply:
Make the imports like this?:
```
import functools
from binascii import hexlify
import time
import unittest

import config
import mle
import node
```

Reply:
I found that `binascii` is not used any more in **Cert_7_1_01**, so I simply removed it.